chain/events/filter/index.go

```diff
@@ -32,6 +32,8 @@ var pragmas = []string{
 	"PRAGMA read_uncommitted = ON",
 }
 
+// Any changes to this schema should be matched for the `lotus-shed indexes backfill-events` command
```

Comment:
It's mainly the queries I'm concerned about; see https://github.com/filecoin-project/lotus/pull/11723 where this becomes a problem and is why I thought to add this comment in there - the schema will change, then the queries will break in lotus-shed. We could put all the queries in index.go, but then we'd have to make them public to make them useful, and do we want to be stuck with additional public APIs just for this? We could, it just seems noisy for a command that basically nobody uses (I was running this on calibnet and it was painfully slow, yet nobody's complained since it was introduced as far as I know).